common/src/main/java/whocraft/tardis_refined/client/TardisClientData.java

```diff
@@ -322,6 +326,24 @@ public void tickClientside() {
             }
 
 
+
+            if (isThisTardis && tardisLevel.getGameTime() % nextFemaleAmbientNoiseCall == 0) {
+                nextFemaleAmbientNoiseCall = tardisLevel.random.nextInt(6000, 36000);
+                LocalPlayer player = Minecraft.getInstance().player;
+                RandomSource randomSource = tardisLevel.random;
+
+                QuickSimpleSound simpleSoundInstance = new QuickSimpleSound(SoundRegistry.INTERIOR_VOICE.get(), SoundSource.AMBIENT);
```

Comment:
Can this be cached somewhere so we aren't constructing the object everytime? Or if it needs to be dynamic, move this into a method so it's more reusable and easier to debug.

Reply:
Cached